### ISSUE TITLE
GUI enabled in inactive worlds

### ISSUE THREAD

**Issue by SlimeDog** [PaulAnthonyReitz/ReitzMMO#4]
Spigot 1.14.1
ReitzMMO 1.14

```
# This config is used to set all world levels configurations.
#  Use -1 to make the plugin disabled on a world
world: -1
world_nether: -1
world_the_end: -1
```
but GUI is accessible in all worlds.

**Comment by RitzDaCat** [comment]
https://github.com/PaulAnthonyReitz/ReitzMMO/commit/945ac5dce1bd52c38e2dff9a6e7ec92e7bcf7e53

**Comment by SlimeDog** [comment]
Confirmed fixed in v 1.15. Thanks.
